task3/task3/Program.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace task3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
```

Comment:
Здесь нужно разместить запрос числа.
```suggestion
        {
            Console.WriteLine("Введите число x");
            double n = double.Parse(Console.ReadLine());
```